source/bin/tvm.rs

```diff
@@ -170,3 +180,4 @@ mod tests {
         );
     }
 }
+
```

Comment:
It's just more clean, and even used in some coding standards (cc with `-std=c89 -pedantic`).